### DISCUSSION TITLE
File/Folder icons not showing

### DISCUSSION THREAD

**Discussion by anwesh-b** [NvChad/NvChad#2252] [Q&A]
I believe `nvim-web-devicons` sets the icon for differnt files and folders.
I setup the NvChad, and the files and folder icons are still missing.
![image](https://github.com/NvChad/NvChad/assets/45763486/d551e8a4-8ec8-4e8e-9582-c0c5be58a93d)
Is there anything I am missing out? 

OS: Fedora Linux 38
GNOME Version: 44.3
Windowing System: Wayland
Kernel Version: Linux 6.4.4-200.fc38.x86_64

**Comment by siduck** [comment]
@anwesh-b update your nerdfonts
